appengine/flexible_python37_and_earlier/hello_world_django/README.md

```diff
@@ -0,0 +1,63 @@
+# Django sample for Google App Engine Flexible Environment
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=appengine/flexible/hello_world_django/README.md
+
+This is a basic hello world [Django](https://www.djangoproject.com/) example
+for [Google App Engine Flexible Environment](https://cloud.google.com/appengine).
+
+## Running locally
+
+You can run locally using django's `manage.py`:
+
+    $ python manage.py runserver
+
+## Deployment & how the application runs on Google App Engine.
+
+Follow the standard deployment instructions in
+[the top-level README](../README.md). Google App Engine runs the application
+using [gunicorn](http://gunicorn.org/) as defined by `entrypoint` in
+[`app.yaml`](app.yaml). You can use a different WSGI container if you want, as
+long as it listens for web traffic on port `$PORT` and is declared in
+[`requirements.txt`](requirements.txt).
+
+## How this was created
+
+This project was created using standard Django commands:
+
+    $ virtualenv env
```

Comment:
Ditto for `virtualenv`

Reply:
I added a link to https://cloud.google.com/python/docs/setup. I think it still make sense to keep these historical commands. Their goal was to show how that sample app was created.

Reply:
Ah. I skimmed through and didn't read properly! Sorry about that. Sounds good to keep it as is.